admin/inc/functions_themes.php

```diff
@@ -1510,6 +1479,7 @@ function upgrade_css_120_to_140($css)
 		}
 	}
 
+	// MD5 is not the safest method to use.
```

Comment:
What is the point of this comment? These are not passwords so md5 is perfectly fine here.

Reply:
I know these are not passwords;  But still, its not safe to use MD5 as its [easily unhashable](https://www.google.com/search?client=safari&rls=en&q=unhash+md5&ie=UTF-8&oe=UTF-8)